_posts/2023-08-03-organisers-two-fer-typescript-tailwind.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "TypeScript Tales and Tailwind Types"
+summary: "An organisers' two-fer covering two advanced tools for building the web"
+date: 2023-08-03T19:15:00
+meetup: https://www.meetup.com/async-web-tech-meetup/events/295080379/
+speakers:
+  - name: Jake 'Sid' Smith
+    link: https://twitter.com/JakeSidSmith
+  - name: Yann Eves
+    link: https://yannev.es
+sponsors:
+image:
+  url:
+  title: Tailwind and TypeScript logos
+tags:
+  - programming
+  - javascript
+  - typescript
+  - tailwind
+  - css
+venue:
+  name: Runway East
+  link: https://runwayea.st/locations/brighton?utm_source=external&utm_medium=event&utm_campaign=sponsorship
+  location: https://goo.gl/maps/WGe2p9D7Wup3LdNt8
+  address: York And Elder Works, 50 New England St, Brighton BN1 4AW
+  latlong: 50.8325788,-0.1420808
+layout: event.html
+collection: events
+---
+
+**NEW VENUE:** We're trialling a move to [Runway East](https://runwayea.st/locations/brighton?utm_source=external&utm_medium=event&utm_campaign=sponsorship), a brand new space we think makes an even better long-term home for Async meetups. Located a short walk from Brighton railway station on the ground floor of the new York & Elder development.
+
+This month we bring you a special organisers' two-fer presented by Jake and Yann!
+
+#### The Power of TypeScript
+
+Jake will speak on advanced patterns in TypeScript you'll be glad to know. Along with powerful uses you may wish you didn't see. `<T>`rigger warning.
+
+#### Picking up Tailwind
```

Comment:
Would this read better having the lightning talk first? (assuming you'll be speaking first)